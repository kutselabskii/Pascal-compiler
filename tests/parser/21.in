program abc;

var a: real;
begin
	begin
		a := 5;
		a := 2.5;

end.