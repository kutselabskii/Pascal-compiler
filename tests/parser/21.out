Line 14 : Col 5 : Parser error :: 'end' expected
